.github/workflows/linux-BUILD_DEPS.yml

```diff
@@ -13,29 +13,33 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        use_ortools_stable : ["OFF"]
 
     steps:
     - uses: actions/checkout@v2 
 
     - name: Install libraries
       run: |
            sudo apt-get update --fix-missing
-           sudo apt-get install libuuid1 uuid-dev libssh2-1 libssh2-1-dev libidn2-0 libidn2-dev libidn11 libidn11-dev gtk2.0 libb64-dev       
+           sudo apt-get install libuuid1 uuid-dev libssh2-1 libssh2-1-dev libidn2-0 libidn2-dev libidn11 libidn11-dev gtk2.0 libb64-dev
+           
+    - name: Download pre-compiled librairies
+      run: |
+           wget https://github.com/AntaresSimulatorTeam/antares-deps/releases/download/v1.0.0/rte-antares-deps-${{matrix.os}}-Release-all.tar.gz
+           tar -xvf rte-antares-deps-${{matrix.os}}-Release-all.tar.gz
+           rm -rf rte-antares-deps-${{matrix.os}}-Release-all.tar.gz
+           
+    - name: Init submodule
+      run: |
+           git submodule update --init --recursive src
 
     - name: Configure
       run: |
-           cmake -B _build -S src -DBUILD_DEPS=ON -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON -DUSE_ORTOOLS_STABLE=${{ matrix.use_ortools_stable }}
-    
-    - name: Remove dependency compilation directory
-      run: |
-           rm -rf _build/build_deps
+           cmake -B _build -S src -DDEPS_INSTALL_DIR=./rte-antares-deps-Release -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON -DBUILD_OPENSSL=ON -DBUILD_curl=ON -DBUILD_wxWidgets=ON
```

Comment:
Need hack with `-DBUILD_OPENSSL=ON -DBUILD_curl=ON -DBUILD_wxWidgets=ON` to be able to used precompiled dependencies.
In Ubuntu find_package(wxWidgets) doesn't find wxWidgets if install compiled librairy is moved, wxWidgets is still built in this CI.